src/schema/rules/sidecars/pet.yaml

```diff
@@ -150,12 +150,16 @@ PETReconstruction:
       description_addendum: This partly matches the DICOM Tag (0054,1103) `Reconstruction Method`.
     ReconMethodParameterValues:
       level: required
+      level_addendum: |
+        Required if `ReconMethodParameterLabels` does not contain 'none', optional if
+        no labels exist.
       description_addendum: This partly matches the DICOM Tag (0054,1103) `Reconstruction Method`.
     ReconFilterType:
       level: required
       description_addendum: This partly matches the DICOM Tag (0018,1210) `Convolution Kernel`.
     ReconFilterSize:
       level: required
+      level_addendum: Required if `ReconFilterType` is not 'none'.
       description_addendum: This partly matches the DICOM Tag (0018,1210) `Convolution Kernel`.
```

Comment:
I think this needs to be split out into a separate rule as well.

Reply:
I'm sorry, I don't understand what you mean by that. Is that not what is defined below in `EntitiesReconFilterData`?